### DISCUSSION TITLE
Share Your Website! 🌎

### DISCUSSION THREAD

**Discussion by khoa-l** [hack4impact-calpoly/starter-pack#2] [Show and tell]
**This is not the Final Submission. 
If you are submitting for completion, go to the Final Submission discussion post!**

Post a link to your website below!
[Visit the example one here!](https://khoa-l.github.io/)

**Comment by TeachMeTW** [comment]
I've created a website based on the starter pack, I made it steins;gate themed because why not?
[Website](https://hack4impact-starter.vercel.app/index.html)
[The Repo/Sauce](https://github.com/TeachMeTW/hack4impact_starter)




I already had my personal website, but did include stuff that I forgot to which the starter pack reminded me about (resume, github, linkedin).
It was a great resource, I love the structure and overall layout -- and the info too.

[My website (it does lag for a bit while loading due to my model that I might remove)](https://robinttw.com)

**Reply by khoa-l** [reply]
Awesome work! I can already feel John Titor throwing me into an Okabe-esque time loop. Would you make a submission in the **Submissions** discussion group and also link your GitHub repository for the site 👀 :D

The instructions are on the last Lab of the Starter Pack!

**Comment by munozAndrew** [comment]
https://munozandrew.github.io/PortfolIoHACK4IMPACT/   Im currently on lab 28 and making progress

**Comment by devinhadley** [comment]
Here is a [link](https://devinhadley.github.io/) to my website. I'm enjoying reviewing the fundamentals of web design so far!

**Comment by RyanSchmit** [comment]
This is a link to my [website](https://ryanschmit.github.io/). I have been learning a lot from this starter pack. I am currently on lab 28.

**Reply by khoa-l** [reply]
Top-tier photo choice. Love the lighting!

**Comment by charlesmoreno32** [comment]
Here's my [website.](https://charlesmoreno32.github.io/) I had some trouble setting up the github stuff but was eventually able to figure everything out. I also just am finishing with lab 28. Other than that I feel pretty confident in what I've done but would probably need more usage and practice to completely ingrain HTML and Git into my skillset.

**Comment by sapierre26** [comment]
Here's my website! file:///C:/Users/sgpie/.vscode/index.html 
I'm feeling good about using HTML, but I'm struggling with the connection between GitHub and Visual Studio Code.

**Reply by khoa-l** [reply]
What sort of issues are you experiencing? If you make a post, we could definitely try to help you out!

**Reply by sapierre26** [reply]
I was struggling with the remote connection lab. Being able to update my GitHub through the terminal in VSC.

**Comment by SilveerDusk** [comment]
Having fun making my [website](https://silveerdusk.github.io/). I threw in a few jokes to fill up the blank space while I work on the Starter Kit. Let me know if any of them are prime jokes!

**Comment by Sriya-Adunur** [comment]
Here's the link to my website: [https://sriya-adunur.github.io](https://sriya-adunur.github.io)
I initially struggled with creating the link to my website as it showed up an error when I tried to use the https link, and was instead kept as a file. So I decided to redo the whole process again, copying my code from each of the HTML files and it worked! Definitely a long process, but very rewarding! I still need to make my website more visually appealing, but I'm happy with the progress that I've made.

**Comment by evayktv** [comment]
This is my [website](https://evayktv.github.io/index.html). Nothing too crazy, just some slight modifications.

**Comment by Robert303V** [comment]
Here's My Website So Far! 
[My Website](https://r303vermeulen.github.io/)
I still need to add the stylings, but I'm happy with the HTML work I did!

I encourage you to go to my portfolio & check out the Game & Digital Art Website I have!
It's something I made a bit ago, but I've never figured out how to host it remotely for others to see as it's only ever been on my local machine. So now that it's live, it'd love for people to check it out!

**Reply by khoa-l** [reply]
I love the portfolio! The tables are very nice

**Comment by ethantrantalis** [comment]
My website can be found at this [link](https://rob0t3000.github.io/). No major modifications yet

**Reply by khoa-l** [reply]
Hi Ethan! I don't see a link 👀 is that just me?

**Reply by ethantrantalis** [reply]
Yah the link was not working for some reason, i am fixing now

**Comment by HoldenEv** [comment]
Here is the website that I create [https://holdenev.github.io/](url) it was a lot of fun to get a chance to personalize it a little,  I wanted to use this for applications and I did not really have anything to add to the portfolio or the blog sections so I took it out. Hope you enjoy!

**Reply by khoa-l** [reply]
Hey Holden! It seems like your website link doesn't work~

Also it's great that you're looking to personalize it, we're planning on building up those sections later and your portfolio should have just the website on it for now, so make sure they're there when you submit! Keep at it ✨

**Reply by HoldenEv** [reply]
Whoops here is the link  [my websiteI](https://holdenev.github.io/) will make sure to add to sections a restyle them before I use the final submission.

**Reply by khoa-l** [reply]
I love the font you used for your logo!

**Reply by HoldenEv** [reply]
Thank you!

**Comment by lukewaltz** [comment]
heres a link to my website: https://lukewaltz.github.io
Still a work in progress but I look forward to adding some styling in the next couple hours.

**Comment by Brian-Kwong** [comment]
https://brian-kwong.github.io/

**Comment by jmdebarro** [comment]
Hey I'm Jonathan, Currently working on the website but hit the point where you share it. 
It will look better soon
[Website](https://jmdebarro.github.io/)

**Comment by jqueja** [comment]
Hello everyone! This is my website so far at the time of doing the labs.
https://jqueja.github.io/

**Comment by Elmanhands** [comment]
Here's a [link](https://elmanhands.github.io/) to my website! Currently on lab 28 of the starter pack!

**Comment by davism7** [comment]
This is my website: https://davism7.github.io

**Comment by Jhammett03** [comment]
Here is a link to my [website](https://Jhammett03.github.io/main)! I just finished the HTML portion of the starter pack and am ready to work on the CSS!

**Comment by cbgabler** [comment]
Here is my [website](https://cbgabler.github.io/). Working on layout and css.

**Comment by amandayu255** [comment]
Here is a link to my [website](https://amandayu255.github.io/)! I learned a lot about the different tags, forms, attributes, and more.

**Comment by logancostello** [comment]
Welcome to my[Website](file:///Users/logancostello/CPE225/6Assignment/logancostello.github.io/index.html)! It's currently a work in progress but I've been learning (relearning technically) a lot about web design. I've also learned a lot about git and github. In fact this is my first project on github!

**Reply by khoa-l** [reply]
Your link doesn't seem to be working?

**Comment by Kaweees** [comment]
Here is a [link](https://miguelvf.dev/) to my website, featuring the static site generator [Hugo](https://gohugo.io/) and Continuous Integration to build my site from markdown pages and push the built pages to the internet!

**Comment by wesleytam88** [comment]
Here's a link to my website: https://wesleytam88.github.io/
It's still WIP but it's been smooth sailing going through the starter pack, no major issues!

**Comment by JoaquinArredondo** [comment]
Here's my website: https://joaquinarredondo.github.io/
I still want to add way more to it, so far the starter pack was fun to learn.

**Comment by riannalei** [comment]
Hello! this is my webitse [WEBSITE](https://riannalei.github.io/)
still need to fix/add some stuff.

**Comment by NoahMoscovici** [comment]
My current website 😎: https://noahmoscovici.github.io/
Source code: https://github.com/NoahMoscovici/noahmoscovici.github.io

**Comment by jacksonmcl** [comment]
Current Website: https://jacksonmcl.github.io/
Still a work in progress! Need to add my own styling but excited to toy around with it.

**Comment by achou05** [comment]
This is my current website: https://achou05.github.io/

It may be basic but I am proud of it and have lots of work. The starter pack was enjoyable every step of the way, I was amazed and said "this is sick" a handful of times. I definitely need to add my own style.

**Comment by greinhard2003** [comment]
This is my current website: https://greinhard2003.github.io/

I built it using the starter pack as my guide. Its still a work in progress as I still need to make it pretty.

**Comment by mtbosio** [comment]
This is my current website: https://mtbosio.github.io/

I'm on lab 28 of the starter pack. Still a work in progress.

**Comment by Belal-Elshenety** [comment]
This is my current [website](https://belal-elshenety.github.io/)

I've just finished lab 28 of the starter pack. Very exited to see how it will look like when I put it all together!

**Comment by soramicha** [comment]
My current website looks like this https://soramicha.github.io/ It's still very much a work in progress. I am on section lab 28 at the moment.

**Comment by aahmadyar123** [comment]
Hers is my current website: https://aahmadyar123.github.io/
Still have a ways to go!

**Comment by cwong239** [comment]
Here is a link to my website, I'm still working on the CSS, so check back later https://cwong239.github.io

**Comment by anvch** [comment]
Hello! This is my website so far: https://anvch.github.io/
I've been having a lot of fun learning the HTML and personalizing the website!

**Comment by amalekia** [comment]
I am on lab 28 now and here is my progress https://amalekia.github.io/

**Comment by LambNNot** [comment]
Hi, this is the current shape of my site: https://lambnnot.github.io (pretty barebones I know :P). I've yet to add much personal flair since I am working on this very late minute, but I am on Lab 28 now :D

**Comment by IvanA673** [comment]
This is my [website](https://ivana673.github.io/) I hope you like it! I'm planning on cleaning it up once I reach the end of the project. This starter pack has been very fun and informative!

**Comment by aumerov** [comment]
Finished up to lab 28, [**here**](https://github.com/aumerov/aumerov.github.io) is my progress so far! 
I am having an issue with the file structure, which is strange because it looks perfectly fine on my local machine.

**Comment by rachanaraju3** [comment]
Here's the link to my [website](https://rachanaraju3.github.io/)! Just finished lab 28 and plan on personalizing it later. I've learned a lot so far from the starter pack and am very excited to see what my final product will be!

**Comment by achan214** [comment]
This is the [link](https://achan214.github.io) to my website so far! It's very basic right now but hopefully I'll have more time to personalize it later on. Just trying to get it done by today, but I'm learning a lot so far! Onward to lab 29!

**Comment by amalekia** [comment]
Hey guys this is my website: [link](https://amalekia.github.io)
Also my repo: [Repo](https://github.com/amalekia/amalekia.github.io)

I didn't get to do too much to it since I was very loaded this pas week and weekend however I do plan on adding onto it from hereon out!

**Comment by elainalyons** [comment]
Hi! I just finished Lab 28 and look forward to learning more about CSS and how to personalize my website (make it not look sad). So far the starter kit has been really interesting and easy to follow!
My Website: https://elainalyons.github.io/

**Comment by ArjunR27** [comment]
Hey guys I'm almost done with my e-portfolio buyt I'm in need of some help. I can't figure out why my images won't show up on Github Pages. They show up when I follow the relative path. https://github.com/ArjunR27/ArjunR27.github.io --> Here's the github link.

**Comment by poornimagodavarthy** [comment]
hi! just working on lab 28 and this is my progress so far: https://poornimagodavarthy.github.io

**Comment by HenryHorse** [comment]
Hello! Enjoying working on the starter pack, here's my website thus far: https://henryhorse.github.io

**Comment by peterchinh** [comment]
Here's my progress to far: https://peterchinh.github.io
Still needs some work but this starter pack has been really helpful!

**Comment by ryyyvo** [comment]
I forgot to post my progress, but that's what git is for!!
https://ryyyvo.github.io/
Thank you for the clear and concise teachings on the starter pack!

**Reply by khoa-l** [reply]
Hi Ryan, if this is supposed to be a final submission, remember to submit it in the Final Submission discussion post or it won't be considered turned in!

**Comment by kylelin23** [comment]
Here is my website: https://kylelin23.github.io/kylelin.github.io/. 
I will definitely add more to it later!

**Comment by leilat014** [comment]
Here is my website so far: https://leilat014.github.io/.
I had the best time reviewing HTML and learning about how to properly set up Git. I'm super excited to design it more and add some color!

**Comment by SumedhaKun** [comment]
Website: https://sumedhakun.github.io/

**Comment by michaelmurrayiv** [comment]
Here is the [link](https://michaelmurrayiv.github.io) to my website! Currently have only completed the HTML portion of the site, but am excited to get into the styling with CSS!

**Comment by sahkar** [comment]
Link to my website: [Link](https://sahkar.github.io/)
Link to github repo: [Link to repo](https://github.com/sahkar/sahkar.github.io)

Will keep adding to the CSS!

**Comment by nriley-14** [comment]
Here's my [website ](https://nriley-14.github.io/) and [repository](https://github.com/nriley-14/nriley-14.github.io/tree/main) so far! The starter pack has been a great tool to review. I am excited to add some styling to my website.

**Comment by 0felicia0** [comment]
Website: 0felicia0.github.io
Here's the repository: https://github.com/0felicia0/0felicia0.github.io

**Reply by khoa-l** [reply]
Hi, Felicia! It seems to be because your repository is not named after your username~ I think it should be `0felicia0.github.io`. Go to settings and just rename it!

**Reply by 0felicia0** [reply]
Thank you!!

**Comment by keilamohan** [comment]
Here is the link to my website! I am excited to build upon it, right now it is somewhat plain :)
https://keilamohan.github.io/

**Comment by xgarcia17** [comment]
Hey everyone! I'm enjoying learning more about web dev so far! You can check out my website here: https://xgarcia17.github.io/

**Comment by palldas** [comment]
Here is my site, I'll keep adding to it! https://palldas.github.io/

**Comment by sab115** [comment]
Here's my link to the website: https://sab115.github.io/

**Comment by Martin-Wise** [comment]
Here's my website! Im definitely going to keep adding and refining it.
https://martin-wise.github.io/

**Comment by ajesus7** [comment]
Here's my website so far! I just reached Lab 28, and will continue to work through the labs to add some styling to the website.

I've really enjoyed the starter pack and its clean, concise layout of information!

https://ajesus7.github.io/starter-pack/

**Comment by diegonieves8** [comment]
Here's my website so far! I really like how it looks right now and with just some tweaks here and there, I'll finish it to submit it for tonight.

https://diegonieves8.github.io

**Comment by stkeopila** [comment]
Here is my Personal Website so far, I really enjoy the process of being able to edit the website at my own free will.

https://stkeopila.github.io

**Comment by ivannghi** [comment]
My personal website is going well, learning a lot, making this website is going to help it stick too!
Link: https://ivannghi.github.io

**Comment by sofijadimitrijevic1** [comment]
My personal website is going great! I'm learning so much from this starter pack and I'm so grateful to Hack4Impact for taking the time to make this amazing tutorial! Can't wait to add more!! My website link: [https://sofijadimitrijevic1.github.io/](url)

**Comment by YaseminAkkaya** [comment]
Here is my website so far! Still need to add some more styling and content: https://yaseminakkaya.github.io

**Comment by Sam2545** [comment]
Here is my [website](https://sam2545.github.io/) so far! I have reached lab 28 and I'm looking to give my website a futuristic tone using CSS

**Comment by alec-cheng3** [comment]
Here is the link to the website: https://alec-cheng3.github.io/
It was cool to finally learn how to use git. Been struggling for so long. HTML was an interesting language to learn. Wasn't really sure what to expect but I enjoyed learning it.

**Comment by jeremiahliao** [comment]
work in progress: https://jeremiahliao.github.io

**Comment by ktaschek** [comment]
Hey All! This is my finished website: https://ktaschek.github.io/
It was super fun to learn some CSS and HTML!

**Comment by FlowerCake13** [comment]
Hi! I'm still working on it, but here is what I have so far for my website!
[Website](https://flowercake13.github.io/hack4impact-starter/)
[Repository](https://github.com/FlowerCake13/hack4impact-starter)

**Comment by thienan11** [comment]
Currently working on my website, just need to add some styling and it'll be good to go! Check it out!
[website](https://thienan11-hack4impact-website.vercel.app)

**Comment by seanmnguyen** [comment]
Here's a link to my [website](https://seanmnguyen.github.io/)! It's my first time working with HTML, and I'm learning a lot. Can't wait to stylize it a bit more in the next few labs!

**Comment by whiniy** [comment]
Link to my website so far! https://whiniy.github.io/

**Comment by ishwaryanam** [comment]
This is what my website currently looks like: 
https://ishwaryanam.github.io/

**Comment by kyramirez** [comment]
website: https://kyramirez.github.io/ 
Here is my website!!

**Comment by Antonio0913** [comment]
this is my [website](https://antonio0913.github.io/) nothing too exciting yet

**Comment by Kyan73** [comment]
Website link: https://kyan73.github.io/

I enjoyed learning more about HTML!

**Comment by mio-naka** [comment]
Website link: https://mio-naka.github.io/
Enjoyed making a website with HTML!

**Comment by idavoong** [comment]
Still working on my website but here's the link: https://idavoong.github.io/

**Comment by gracebw7** [comment]
Here is the [link](https://gracebw7.github.io/) to my website. Still working on it though!

**Comment by rcereno** [comment]
Hello! Here is the link to my [website](https://rcereno.github.io/). Starter Pack is going well so far. Currently on Lab 28 and having fun going through it.

**Comment by tammy-si** [comment]
Hey I'm currently on Lab 28. Here's the link to my [website](https://tammy-si.github.io).

**Comment by Eman-Gon** [comment]
Hello here's my website!
file:///Users/emanschool/Desktop/Eman-Gon%20github.io/index.html

**Comment by ishavarrier** [comment]
Hello, here is my website!
https://ishavarrier.github.io/-ishavarrier-.github.io/

**Comment by Bolt259** [comment]
Hey here's my [website!](https://Bolt259.github.io)
